lang/luafilesystem/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=luafilesystem
-PKG_VERSION:=1.7.0.2
+PKG_VERSION:=1.8.0
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_MIRROR_HASH:=f109017711dd77d74a39bd1c0975be7edf726da0265f16496ea468481fac0b85
+PKG_MIRROR_HASH:=16d17c788b8093f2047325343f5e9b74cccb1ea96001e45914a58bbae8932495
 PKG_SOURCE_URL:=https://github.com/keplerproject/luafilesystem.git
-PKG_SOURCE_VERSION:=de87218e9798c4dd1a40d65403d99e9e82e1cfa0
+PKG_SOURCE_VERSION:=7c6e1b013caec0602ca4796df3b1d7253a2dd258
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
This is already default.

Reply:
yes, but it's completely out of scope of this PR, and it would be wrong to make unrelated changes under the banner of a version bump

Reply:
So when do you propose that this be fixed?

Reply:
at anyone's leisure, but not as part of this commit.  It does zero harm in anyway, it's simply redundant.